src/templates/tsconfig.json

```diff
@@ -12,12 +14,10 @@
 		],
 		"module": "umd",
 		"moduleResolution": "node",
-		"noImplicitAny": true,
-		"noImplicitThis": true,
```

Comment:
I found I added `"noUnusedLocals": true` to my create app, worth considering here?

Reply:
good idea, that will be useful.

Reply:
done